react-game/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "react-game",
+  "version": "0.1.0",
+  "homepage": ".",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.9",
```

Comment:
Use devDependencies block to store packages which you need for building and testing application